src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -838,7 +859,7 @@ internal WorkspaceModel CreateCustomNode(string name, string category, string de
             };
             var workspace = new CustomNodeWorkspaceModel(info, nodeFactory);
 
-            RegisterCustomNodeWorkspace(workspace, isTestMode);
+            RegisterCustomNodeWorkspace(workspace);
```

Comment:
This function `CreateCustomNode` should not have the `bool isTestMode = false` argument.

Reply:
thanks will remove.